File: sbe/okx/okx_sbe_1_0/generated-python/com_okx_sbe/SourceEnum.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple, Set
from io import BytesIO

class SourceEnum:
    """Enum values."""
    NORMAL = 0
    ELP = 1
    NULL_VALUE = -128

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/okx/okx_sbe_1_0/generated-php/com_okx_sbe/SourceEnum.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class SourceEnum
{
    public const NORMAL = 0;
    public const ELP = 1;
    public const NULL_VALUE = -128;

    public static function encodedLength(): int
    {
        return 1;
    }
}
